feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go

```diff
@@ -400,8 +401,9 @@ func verifyChassisIsAncestor(t *testing.T, dut *ondatra.DUTDevice, comp string)
 			t.Errorf("Chassis component NOT found as an ancestor of component %s", comp)
 			break
 		}
-		got := gnmi.Get(t, dut, gnmi.OC().Component(val).Type().State())
-		if got == chassisType {
+		gotV := gnmi.Lookup(t, dut, gnmi.OC().Component(val).Type().State())
+		got, present := gotV.Val()
+		if present && got == chassisType {
```

Comment:
This change was not included in the PR description / not part of the deviation, it appears

Reply:
Thanks. Updated the PR description.